### PR TITLE
Russian

### PR DESCRIPTION
**Pull request by ancouli**
Will these changes allow us to publish the website in Russian? 

I am not happy with my changes in get_langs() (commenting out the code). But I didn't want to lose the existing code in case we want to add an incomplete language in the future.

### CODE REVIEW

**Comment by xi** [comment]
> I am not happy with my changes in get_langs() (commenting out the code). But I didn't want to lose the existing code in case we want to add an incomplete language in the future.

Better remove the code instead of commenting it out. That is what we have git for.

File: langs/ru.ini [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+legal="правовые информации"
+email="адрес электронной почты"
 fax="Факс"
 telephone="Телефон"
 bus="Автобус"
 subway="Метро"
+skip="перейти к содержанию"
+newsletter="подписаться на рассылку"
+subscribe="подписаться"
+map="план города"
```
Comment (xi) [inline langs/ru.ini:17]: Did you do these manually? The easiest way is to use [transifex-client](https://pypi.org/project/transifex-client/)

**Comment by ancouli** [comment]
Okay! Should I also remove the parameter in get_langs() and adapt the call of the function in admin/index.php accordingly since the parameter is not used at the moment?

File: langs/ru.ini (hunk 1 above)
Comment (ancouli) [reply langs/ru.ini:17]: Thanks for the tip!

**Comment by xi** [comment]
> Should I also remove the parameter in get_langs()

Not sure. On the one hand we could leave it in in case we ave a similar case again in the future. On the other hand it might be confusing. I see arguments for both sides, so I will let you decide.

**Comment by ancouli** [comment]
> Not sure. On the one hand we could leave it in in case we ave a similar case again in the future. On the other hand it might be confusing. I see arguments for both sides, so I will let you decide.

I decided to keep it for now.

**Pull request merged**